addons/mail/static/src/core/common/message_reaction_list.js

```diff
@@ -0,0 +1,56 @@
+import { Component, markup } from "@odoo/owl";
+import { _t } from "@web/core/l10n/translation";
+
+export class MessageReactionList extends Component {
+    static template = "mail.MessageReaction.List";
```

Comment:
Template name should match component name. No extra `.`.

We try as much as possible to name idiomatically as `[module-name].[ComponentName]`.
The rare case we add another level of `.` is if we have a part of template in its own `t-name`, e.g. `mail.MessagingMenu.content`. To clarify this is a part, this is camelCase rather than PascalCase